src/main/java/sirius/db/mixing/types/BaseEntityRef.java

```diff
@@ -203,6 +203,15 @@ public Optional<E> getValueIfPresent() {
      */
     protected abstract Optional<E> find(Class<E> type, I id);
 
+    /**
+     * Performs the lookup of the entity with the given id using a secondary node of a DB cluster.
```

Comment:
**Optional:** Should mention that this does the same as `find` on db systems that don't support fetching from secondary nodes.